src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1133,19 +978,11 @@ void SolverInterfaceImpl:: mapReadDataTo
       int inDataID = context.fromData->getID();
       int outDataID = context.toData->getID();
       context.toData->values() = Eigen::VectorXd::Zero(context.toData->values().size());
-      //assign(context.toData->values()) = 0.0;
       DEBUG("Map data \"" << context.fromData->getName()
                    << "\" to mesh \"" << context.mesh->getName() << "\"");
       assertion(mappingContext.mapping==context.mappingContext.mapping);
       mappingContext.mapping->map(inDataID, outDataID);
-#     ifndef NDEBUG
-      int max = context.toData->values().size();
-      std::ostringstream stream;
-      for (int i=0; (i < max) && (i < 10); i++){
-        stream << context.toData->values()[i] << " ";
-      }
-      DEBUG("First mapped values = " << stream.str());
-#     endif
+      DEBUG("First mapped values = " << context.toData->values());
```

Comment:
Why changing this? This was always very helpful to not spam the complete debug output.